.github/workflows/release.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     needs:
       - build_linux
       - build_macos
-      - build_windows
```

Comment:
Unless I'm missing something, this diff is broken 🤔 , this isn't what's currently on trunk...

Reply:
maybe LCA related